### PR TITLE
[13.0][IMP] hr_employee_calendar_planning: set calendar_ids from Create employee button from user

### PR DESCRIPTION
**Pull request by victoralmau**
Backport from 14.0:
- https://github.com/OCA/hr/commit/8d716ba6bb4d0fe9930a735dfa9074a24239303a
- https://github.com/OCA/hr/commit/7e36cd0002a14999d0fa88f6a1de234fe11da4fc
- https://github.com/OCA/hr/commit/e609d526b45b61b1afe203cd2bf30b6277e4e075
- https://github.com/OCA/hr/commit/ad76a68f63c55a0eb4ec1c75b52a08aae4e5bcd3

Please @pedrobaeza and @chienandalu can you review it?

@Tecnativa TT44093

### CODE REVIEW

File: hr_employee_calendar_planning/models/resource_calendar.py [hunk 1]
```diff
@@ -11,12 +11,18 @@ class ResourceCalendar(models.Model):
 
     active = fields.Boolean(default=True)
     auto_generate = fields.Boolean()
+    employee_calendar_ids = fields.One2many("hr.employee.calendar", "calendar_id")
```
Comment (pedrobaeza) [inline hr_employee_calendar_planning/models/resource_calendar.py:14]: Why do you need this?

**Review by pedrobaeza — APPROVED** [review]

**Comment by victoralmau** [comment]
Ping @chienandalu

**Review by chienandalu — APPROVED** [review]

**Review by etobella — APPROVED** [review]
:+1:

**Pull request merged**